cves/CVE-2010-1452.yml

```diff
@@ -171,8 +179,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
```

Comment:
Is this the only lesson application? it sounds like there could be potential for other ones.

Reply:
I'm not sure anything else can be applied. This was a simple case of forgetting to check if a pointer was `NULL`. Any suggestions?